docker-compose.yml

```diff
@@ -219,6 +219,14 @@ services:
     - ACCEPT_EULA=Y
     - SA_PASSWORD=Strong!Passw0rd
 
+  # this docker image doesn't work on arm64. It can still be tested on Mac using colima, see https://github.com/abiosoft/colima
+  oracle:
+    image: container-registry.oracle.com/database/free:latest
```

Comment:
How big is this image? 😅 We'll need to update our VMs I suspect, otherwise we'll get pull errors and slow down

Reply:
it's, uh, pretty insane actually.
The compressed size that I get from `docker manifest inspect container-registry.oracle.com/database/free:latest | grep "size"` is 2.4GB,
and the actual image size that I get from `docker inspect -f "{{ .Size }}" container-registry.oracle.com/database/free:latest` is a whopping 8.1GB

Reply:
😱 good lord, that's _definitely_ going to have an impact on the test stage I assume 😬 I guess we should update the VMs _before_ we merge this, to avoid slowing them all down?

Reply:
Yes you're probably right, how do we proceed here ? Do you want me to open a ticket for that ?

Reply:
> Yes you're probably right, how do we proceed here

Well, in reality we need to update the VMs first before merging this, assuming we _want_ to run these tests in CI... which means it's just priorities 😅

Reply:
how about we just disable those tests in CI for now, while _someone_ figures it out ? At least we have the added value of being able to test Oracle changes locally.
Is it possible to mark the test as skipped for the CI, but still have it playable with `BuildAndRunIntegrationTests` without having to hack some checks ?

Reply:
> how about we just disable those tests in CI for now, while someone figures it out?

That's fine by me 

> Is it possible to mark the test as skipped for the CI, but still have it playable with BuildAndRunIntegrationTests without having to hack some checks?

Unfortunately, not easily 🙁 That's a reasonable feature request though 😅 It should be _relatively_ simple to implement - add a trait to the test, and filter out those tests by default in Nuke... I'll see if I can throw something together shortly

Reply:
https://github.com/DataDog/dd-trace-dotnet/pull/5756